packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -472,8 +484,8 @@ export class KeybindingWidget extends ReactWidget {
 
     /**
      * Compare two strings.
-     * @param a {string | undefined} the first string.
-     * @param b {string | undefined} the second string.
+     * @param a the optional first string.
+     * @param b the optional second string.
```

Comment:
would be nice if the "return" is also documented

Reply:
Thank you! I've updated the documentation to be more thorough as suggested.